source/refined-github.css

```diff
@@ -1,6 +1,7 @@
 :root {
 	--rgh-green: var(--fgColor-success, var(--color-success-fg, #1a7f37));
 	--rgh-red: var(--fgColor-danger, var(--color-danger-fg, #cf222e));
+	--rgh-yellow: var(--fgColor-attention, #c69026);
```

Comment:
Yeah this looks correct. Can you also find the second variable name? It might still be used on GitHub enterprise

Reply:
I'm not sure on the second variable name (See beginning of the OP, or the quoted line below). I wasn't able to find any CSS classes in github's served files that matched the naming scheme.

I also don't have access to any enterprise instances to check those.

> 1. Existing CSS vars for `--rgh-green` and `--rgh-red` reference `--color-*` vars that I'm unable to find in refined-github or in github's own CSS files, so I cannot apply an applicable one to the `--rgh-yellow` var I've added.